Registry/AssetProcessorPlatformConfig.setreg

```diff
@@ -172,6 +172,9 @@
                 "Exclude Build": {
                     "pattern": ".*/[Bb]uild/.*"
                 },
+                "Exclude Install": {
+                    "pattern": ".*/[Ii]nstall/.*"
+                },
```

Comment:
This is correct, it will prevent files within any  `[Ii]nstall` directory from being being scanned when by the AssetProcessor